src/PrestaShopBundle/Translation/Provider/ThemeProvider.php

```diff
@@ -219,4 +225,36 @@ public function getThemeCatalogue()
 
         return $this->getCatalogueFromPaths($path, $this->locale, current($this->getFilters()));
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefaultCatalogue($empty = true)
+    {
+        $defaultCatalogue = new MessageCatalogue($this->getLocale());
+
+        foreach ($this->getFilters() as $filter) {
+            $filteredCatalogue = $this->getCatalogueFromPaths(
+                array($this->getDefaultResourceDirectory()),
+                $this->getLocale(),
+                $filter
+            );
+            $defaultCatalogue->addCatalogue($filteredCatalogue);
+        }
+
+        if ($empty) {
+            $defaultCatalogue = $this->emptyCatalogue($defaultCatalogue);
+        }
+
+        return $defaultCatalogue;
+    }
+
+    /**
+     * {@inheritdoc}
+     * string Path to app/Resources/translations/{locale}
+     */
+    public function getDefaultResourceDirectory()
+    {
+        return $this->defaultTranslationDir.'/'.$this->locale;
```

Comment:
To avoid future portability problems, please use `DIRECTORY_SEPARATOR` instead of `'/'`.